simtools/util/model.py

```diff
@@ -57,3 +59,27 @@ def getTelescopeSize(telescopeType):
     else:
         logger.warning('Invalid telescopeType {}'.format(telescopeType))
         return None
+
+
+def validateModelParameter(parNameIn, parValueIn):
```

Comment:
The function to validate a model parameter will always be useful. A user may want to changeParameter, for example, and it would be good to be able to raise an error at this moment, before sim_telarray crashs because of a wrong type.

Also, this validation is not only about type, but about names as well. We change names like fadc_MHz to fadc_mhz, in order to keep the names consistent internally.
 
In the future we might use the DB to read the proper type while validating, and of course, the parameters that come directly from the DB won't require validation. But by now I think this validation is fine.

Reply:
OK